install.sh

```diff
@@ -437,6 +446,10 @@ do_install() {
 				if ! is_dry_run; then
 					set -x
 				fi
+				# install the correct cli version first
+				if [ -n "$cli_pkg_version" ]; then
+					$sh_c "$pkg_manager install -y -q docker-ce-cli-$cli_pkg_version"
```

Comment:
@seemethere a = instead of - before $cli_pkg_version ?

Reply:
I'm just checking here if it's non-empty

Reply:
I think he meant;

```suggestion
					$sh_c "$pkg_manager install -y -q docker-ce-cli=$cli_pkg_version"
```

Reply:
Oh for apt it's an equals sign not a dash

Reply:
Yes, I missed it as well (good 👀 @StefanScherer 🤗)

Reply:
Wait sorry, lost track of where we were in the code, a `-` is the right character for `yum/dnf` install

Reply:
🤦‍♂️ ah, you're right. Perhaps we should change `$pkg_manager` into `$yum` or `$yum_or_dnf` 😅

Reply:
Ah ok, didn't saw that from the phone that this is in the centos case. 🔍